bindings/wasm/crate/src/lib.rs

```diff
@@ -90,7 +95,52 @@ pub fn create_inbound_session(
         .set(receiving_instance);
 
     match result {
-        Ok((_, plaintext)) => Ok(plaintext),
+        Ok((session_id, ciphertext_json)) => Ok(vec![
+            JsValue::from_str(&session_id),
+            JsValue::from_str(&ciphertext_json),
+        ]
+        .into_boxed_slice()),
+        Err(e) => Err(JsValue::from_str(&e.to_string())),
+    }
+}
+
+#[wasm_bindgen]
+pub fn encrypt_message(
+    sending_handle: &str,
+    session_id: &str,
+    message: &str,
+) -> Result<String, JsValue> {
```

Comment:
These types look both to be Strings, only the error gets converted to a wasmbind:JsValue. Shouldn't they be the same type?